web/src/test/java/com/graphhopper/application/resources/RouteResourceCustomModelTest.java

```diff
@@ -66,7 +66,7 @@ private static GraphHopperServerConfiguration createConfig() {
                 putObject("custom_areas.directory", "./src/test/resources/com/graphhopper/application/resources/areas").
                 putObject("import.osm.ignored_highways", "").
                 putObject("graph.encoded_values", "car_access, car_average_speed, road_access, " +
-                        "bike_access, bike_priority, bike_average_speed, bike_road_access, " +
+                        "bike_access, bike_priority, bike_average_speed, bike_road_access, bike_network, " +
```

Comment:
@karussell Depending on the answer on my comment to the changes in the file bike.json the same changes might be needed here as well.